README.md

```diff
@@ -1,61 +1,84 @@
 
-#### Docker
+Then you can run the app directly in the terminal.
+```
+./dungeon-revealer-linux
+```
+Then go to `localhost:3000` in your browser and point your players to `<YOUR_IPADDRESS>:3000`.
+This information is also present in the terminal window.
+
+
+##### OSX
+Open the zip file and extract the files to your preferred location.
+
+Double click the app. A terminal will open with useful information.
+Then go to `localhost:3000` in your browser and point your players to `<YOUR_IPADDRESS>:3000`.
+This information is also present in the terminal window.
+
+##### Windows
+Double click the app. A command prompt will open with useful information.
+Then go to `localhost:3000` in your browser and point your players to `<YOUR_IPADDRESS>:3000`.
+This information is also present in the command prompt window.
+
+##### Docker
 
-To create an instance, run the following:
+
+To create a docker instance, run the following:
 
 ```
 docker pull apclary/dungeon-revealer:latest
 docker run -e DM_PASSWORD=<password> -p <PORT>:3000 -v <DATA_DIR>:/usr/src/app/data -d apclary/dungeon-revealer:latest
 ```
 
 - Replace `<password>` with your chosen DM password
-- Replace `<PORT>` with your preferred port. 
+- Replace `<PORT>` with your preferred port.
 - `<DATA_DIR>` is the directory on the host filesystem in which you want to store the maps and settings. `<DATA_DIR>` **must be an absolute path.** One way to achieve this in linux is to navigate to the directory you want in the terminal and then use `$PWD/data` as `<DATA_DIR>`.
 
 
 In your browser, go to `<YOUR_IPADDRESS>:<PORT>`.
 
 ### Using the app
+The app is separated into two sections. One for the dungeon master and one for the players.
+
 
 #### Dungeon Master
 
-To use dungeon-revealer, the game master and the players must be on the same local network (usually a wifi network). The game master will start the server, navigate to the server's URL in a browser like Chrome, click on the Dungeon Master link, and then enter a password if set. At this point, they will be prompted to upload an image file of the map to share with the other players. The other players will navigate to the server using their own browsers (laptop, tablet, or phone) and will remain at the home page. The connection information is displayed in command prompt for convenience.
+To use dungeon-revealer, the game master and the players must be on the same local network (usually a wifi network). The game master will start the server (see Installation), navigate to the server's URL in a web browser, click on the Dungeon Master link, and then enter a password if it is set. At this point, they will be prompted to upload an image file of the map to share with the other players. The other players will navigate to the server using their own browsers (laptop, tablet, or phone) and will remain at the home page. The connection information is displayed in command prompt for convenience.
```

Comment:
I think it would be useful to make a video or gif of this process.

Reply:
An interactive demo would be awesome, paired with a landing page! I will see what I can do.

Reply:
Let's postpone a landing page for now and start getting the readme ready. @maxb2 Is there anything left that must be addressed before merging this?